<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pago</class>
 <widget class="QDialog" name="Pago">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>329</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Pago{
	background-color: rgb(255, 178, 21);}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="img">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="IMG/imags.qrc">:/iconos/cobrar.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Forma de Pago</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="efectivo_radioButton">
            <property name="text">
             <string>Efectivo</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="tarjeta_radioButton">
            <property name="text">
             <string>Tarjeta</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="pagoWidget">
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>240</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#page{

	background-color: rgb(72, 72, 72);
}
#page_2{

	background-color: rgb(72, 72, 72);
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Calisto MT</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Monto a pagar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="monto">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font-size:14px;</string>
            </property>
            <property name="text">
             <string>$   </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Calisto MT</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Cantidad:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>    $    00.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="img_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>168</y>
           <width>321</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Calisto MT</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="IMG/imags.qrc">:/imagenes/billetes.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>20</y>
           <width>141</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Calisto MT</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Tipo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>MasterCard</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Visa</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Otra</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>301</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Calisto MT</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="placeholderText">
             <string>Ej Marco Antonio Martinez Cruz</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="img_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>140</y>
           <width>301</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Calisto MT</family>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="IMG/imags.qrc">:/imagenes/tarjeta.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>151</width>
           <height>51</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Calisto MT</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Monto a pagar:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="monto_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font-size:14px;</string>
            </property>
            <property name="text">
             <string>$  </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pagar_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-width: 0px;
border-radius: 7px;
background: rgba(0, 143, 57,0.7);

}

QPushButton:hover{
	background: rgba(0, 143, 57,1);
    border-radius: 7px;
	font: 75 9pt &quot;Calisto MT&quot;;
    selection-color: white;  
    padding: 2px 4px;
}</string>
         </property>
         <property name="text">
          <string>Pagar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelar_pushButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-width: 0px;
border-radius: 7px;
background: rgba(255, 50, 57,0.7);

}

QPushButton:hover{
	background: rgba(255, 50, 57,1);
    border-radius: 7px;
	font: 75 9pt &quot;Calisto MT&quot;;
    selection-color: white;  
    padding: 2px 4px;
}</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="IMG/imags.qrc"/>
 </resources>
 <connections/>
</ui>
